packages/dashboard/src/getDefaultConfig.js

```diff
@@ -64,20 +64,28 @@ const getDefaultConfig = () => ({
     weekStartsOn: 1,
   },
   newStoryURL: '',
+  // @todo Should we remove WP-specific props from here?
```

Comment:
@swissspidy Here are a bunch of props that are never used in the `dashboard` package and are in the wp-dashboard package only -- do you happen to know why these were added and if there was a reason for it? Assuming that's also a leftover from moving parts to the `wp-dashboard` package but not sure. Thoughts?

Reply:
Yes these are leftovers from the decoupling work.

Similar to the editor, we'll need to move these to wp-dashboard types.